### ISSUE TITLE
Update to XML 1.12 ipf file -- was: Can't load XML file

### ISSUE THREAD

**Issue by krzywon** [sansigormacros/ncnrsansigormacros#387]
Hi,

I can't load any XML file written in the cansas 1D version 1.1 into the macro.  Every time I try to load it I get this error message:
  root element is not <SASroot> with valid canSAS namespace
  path: "D:Nayomi_Plaza:EQ_SANS:Analysis:1D_data:from_Mantid:"
  root element is not <SASroot> with valid canSAS namespace

Does the macro only reads XML files written in the cansas 1D version 1.0?

THANKS

Nayomi

**Comment by krzywon** [comment]
[srkline] emailed directly  for more information and with options (copy below):

Hi Nayomi,

I'll have to look into which version of canvas standard was used, and what may be different. Could you send me an example data file so I can try it out? I can't guarantee I'll be able to get this fixed anytime soon.

In the meantime, you may want to:
1) convert your data into a different format (maybe Mantid can do this? I don't know)
2) try sasview:
http://sourceforge.net/projects/sasview/files/


Steve

**Comment by krzywon** [comment]
[srkline] Hi Steve,

I think I found the problem.  I had the same issue with the Irena macro, so I contacted Jan Ilavsky.  Pete Jemian mentioned that I needed the 1.12 version of the cansasXML.ipf file to read the new cansas1d format and the version that comes with the macros is 1.11.  

I already got the new file from Jan and replaced the old one.  Now I can import the new XML files into the macros.

Thanks for your suggestions.

Nayomi
